src/pages/home/report/ReportActionsView.js

```diff
@@ -214,6 +212,9 @@ class ReportActionsView extends React.Component {
                 } else if (this.state.newMarkerSequenceNumber === 0) {
                     this.setState({newMarkerSequenceNumber: currentLastSequenceNumber});
                 }
+            } else if (this.state.newMarkerSequenceNumber === 0) {
+                // The report is not in view so we will update the newMarkerSequenceNumber
```

Comment:
Update to the last one so nothing shows as unread?

Reply:
Ah so, the condition here is: 

- report isn't in view
- a new message appeared from someone besides the current user 
- no previous marker is set

So we will update the new marker with this `sequenceNumber`. This basically fixes a bug found by QA here: https://github.com/Expensify/App/issues/10768